### ISSUE TITLE
full-length sequence of circRNA

### ISSUE THREAD

**Issue by Zheng19w** [bioinfo-biols/CIRIquant#53]
Dear Professor,

Hello, thank you for your team's development of CIRIquant, an excellent tool. I have a question that needs your help. I want to know how to obtain the mature full-length sequence of circRNA. Looking forward to your detailed reply!
![image](https://user-images.githubusercontent.com/106860091/228150213-918cd078-feea-4907-829f-e59fa7bd4f91.png)

**Comment by Kevinzjy** [comment]
Hi @Zheng19w , 

Please use our CIRI-full pipeline (https://github.com/bioinfo-biols/CIRI-full/releases) to assemble the full-length circRNA sequence.

**Comment by Zheng19w** [comment]
Thank you for your reply. Can CIRIquant also accurately predict the full length of circRNA? Is there a big difference between the two

**Comment by Kevinzjy** [comment]
> 

No, CIRIquant can not predict the full length of circRNAs. It's not designed for this job.
